drivers/NFSSR.py

```diff
@@ -151,7 +159,7 @@ def probe(self):
 
         self.mount(temppath, self.remotepath)
         try:
-            return nfs.scan_srlist(temppath)
+            return nfs.scan_srlist(temppath, self.dconf.get('server'), self.srcmd.params.get('device_config'))
```

Comment:
We could just pass self.dconf to nfs.scan_srlist here. That has all the parameters we need.